package.json

```diff
@@ -37,6 +37,7 @@
     "rules": {
       "selector-class-pattern": null,
       "max-nesting-depth": 3,
+      "property-no-vendor-prefix": null,
```

Comment:
This needs to be mentioned in the PR doc.

Reply:
Done